static/js/compose_fade.js

```diff
@@ -27,8 +27,8 @@ export function set_focused_recipient(msg_type) {
     };
 
     if (focused_recipient.type === "stream") {
-        const stream_name = $("#stream_message_recipient_stream").val();
-        focused_recipient.topic = $("#stream_message_recipient_topic").val();
+        const stream_name = compose_state.stream_name();
+        focused_recipient.topic = compose_state.topic();
```

Comment:
I believe this tweak changes the trimming behavior; that's probably a good thing in all the cases I see here, but wanted to flag it as a functional change.

Reply:
👍 good to explicitly call out, thanks :)